CMakeLists.txt

```diff
@@ -153,16 +154,20 @@ target_compile_definitions(fms PRIVATE "${_fms_defs_private}")
 
 target_include_directories(fms PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/FMS/include>
                                       $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/FMS/fms>
+                                      $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/FMS/fms2_io/include>
```

Comment:
May I ask which FMS version will be used in this PR? My understand is that we are moving to FMS 2020.04 in early Dec, currently using fms2_io may cause issues in MOM6.

Reply:
The regression tests were run with 2019.01.03, but the changes should work with newer versions as well. I had earlier tested with 2020.02, so I think that is why the fms2_io line is in there. I can remove it for this PR.